plugins/Marketplace/stylesheets/marketplace.less

```
@@ -98,7 +98,7 @@
             padding-bottom: 0;
             margin-bottom: 10px;
             .more {
-                color: @theme-color-text;
+                color: #1976d2;
```

Should be better to use the variable.
```suggestion
                color: @theme-color-link;
```